src/actions/track-chair-actions.js

```diff
@@ -375,3 +385,135 @@ export const reorderProgressFlags =
       dispatch(stopLoading());
     });
   };
+
+/** ********************************************************************************************************* */
+/*                          TEAM LISTS                                                                        */
+/** ********************************************************************************************************* */
+
+export const getSelectionPlans = () => async (dispatch, getState) => {
+  const { currentSummitState } = getState();
+  const accessToken = await getAccessTokenSafely();
+  const { currentSummit } = currentSummitState;
+
+  dispatch(startLoading());
+
+  const params = {
+    page: 1,
+    per_page: 100,
+    access_token: accessToken,
+    expand: "track_groups,track_groups.tracks"
```

Comment:
could we restrict requested data to minimum needed ( fields, relations and expands ) ?